x-pack/plugins/fleet/server/services/agent_policies/full_agent_policy.ts

```diff
@@ -491,13 +490,8 @@ export function transformOutputToFullPolicyOutput(
 
 /**
  * Get id used in full agent policy (sent to the agents)
- * we use "default" for the default policy to avoid breaking changes
```

Comment:
removed this logic, it was added in 8.0: https://github.com/elastic/kibana/pull/111002

@nchaulet do you remember what was the potential breaking change referred here?

Reply:
I think in 7.x we required to have one default output https://github.com/elastic/fleet-server/pull/713 it's probably not relevant anymore